src/styles/utils/_reset.scss

```diff
@@ -0,0 +1,4 @@
+body {
+  margin: 0;
+  background-color: #acdc;
```

Comment:
The background-color value is incorrect. It should be a valid hexadecimal color, RGB, or a named color.